libs/core/src/lib/rimraf-dir.ts

```diff
@@ -1,36 +1,59 @@
 "use strict";
 
 import log from "npmlog";
-import path from "path";
+import path from "node:path";
+import fs from "node:fs";
 import pathExists from "path-exists";
 
 // eslint-disable-next-line @typescript-eslint/no-var-requires
 const childProcess = require("@lerna/child-process");
 
-// NOTE: if rimraf moves the location of its executable, this will need to be updated
-const RIMRAF_CLI = require.resolve("rimraf/bin");
+let rimrafBinPath: string | undefined;
 
-export function rimrafDir(dirPath: string) {
+export async function useRimrafBinPath(): Promise<string> {
```

Comment:
Please can we not export this to make it clear that it is not used outside this file

```suggestion
async function useRimrafBinPath(): Promise<string> {
```

Reply:
good point ^^, sry :see_no_evil: 
I will fix that asap

Reply:
You can commit the suggestion directly in the github UI just FYI

Reply:
oh, wasn't aware of that ^^. Now i have already made a commit from my fork, but good to know, thanks